articles/most-useful-node.js-packages/index.md

```diff
@@ -0,0 +1,515 @@
+## Most Useful Node.js Packages That you need to Know
+
+Open-source frameworks that form the ecosystem of JavaScript back the
+power of Node.js. As a Node.js developer, it is practically inevitable
+to constantly avoid their advantage. With the help of Node.js packages,
+as a developer, you will conclusively save a lot of time and develop
+applications that are light, scalable, high speed with tremendous
+productivity.
+Let\'s have a look at the packages that will assist you to extend your
+inclination to Node.js application development.
+
+### 1. Express
+
+Express.js is the fastest, unopinionated, and simplest web framework forNode.js. It is very flexible designed for building simple page,multipage and hybrid applications that provide a robust set of features for web and mobile applications. The framework is minimal. It is the leading Node.js framework designed for building web applications and
+APIs with robust tooling for HTTP servers. Node.js frameworks such as [`SAILS`](https://www.npmjs.com/package/sails), [`NESTJS`](https://nestjs.com/), [`LOCOMOTIVE`](http://www.locomotivejs.org/) are built on Express. Express is available on the NPM registry.
+
+**Features of Express**
+-   Robust routing.
+-   Focus on high-quality performance.
+-   Superhigh test coverage.
+-   HTTP helpers (such as redirection, caching).
+-   Content negation.
+-   Executable for generating applications quickly.
+
+### 2. Async
+
+Asynchronous is heavily used in Node.js to ensure non-­blocking operations flow. Asynchronous I/O permits other processing to continue even before the first transmission has finished.
+It uses a queue to keep track of tasks, allowing you to append additional tasks, attaching extra callbacks, and handling error inside callbacks, making it a more versatile and robust solution for complex dependency management.
+Async provides several functions that include usual functional such are '`map`', '`filter`', '`reduce`',' `filter`', '`each`' as well as some common patterns for asynchronous control flow('`parallel`' , '`series`' , '`waterfall`').
+
+**Advantages of using Async in your Node.js app include**
+-   It supports inline functions and text strings.
+-   Error handling from the dependency queue.
+-   Use of AsyncLocalStorge within AsyncJs helps to create asynchronous states within Callbacks and promise chain.
+-   A collection of Async functions helps to control the flow through the script.
+-   Helps to integrate AsyncResource with EvenEmitter.
+
+Example of `async/await` in Node.js:
+```js
+async function myFunction(inputValue) {
+    try {
+        const a = await asyncFunc1('value');
+        const b = await asyncFunc2(a);
+        const c = syncFunc3(b);
+        return await asyncFunc4(c);
+    } catch (ex) {
+        // handle exception
+    }
+}
+```
+In this scenario, we can see it is obvious that functions 1,2and 4 are asynchronous.
```

Comment:
You need to include a space after 2. I would take out obvious in case beginner's are reading it that may not immediately understand the example.